src/hll.c

```diff
@@ -47,7 +47,9 @@
 #include "utils/fmgrprotos.h"
 #endif
 #include "utils/guc.h"
+#if (PG_VERSION_NUM < 150000)
```

Comment:
do we need this header at all? e.g. if I switch to PG12 it still compiles fine without it

Reply:
I think we need to keep this header in terms of backward compatibility. See https://github.com/citusdata/postgresql-hll/commit/2d514f2c5da810954089179ce785baddcbdb0d10

Reply:
That referenced commit was created when PG9.3 was the latest version. That is really old.

I confirmed that we can build the project on all supported PG versions today (that is >= PG 10) without having that header.

I suggest we remove the line `#include "utils/int8.h"` altogether.